test/integration/patients/patient/archive.js

```diff
@@ -652,7 +655,10 @@ context('patient archive page', function() {
       })
       .visitOnClock(`/patient/archive/${ testPatient.id }`, { now: testTs() })
       .wait('@routePatient')
-      .wait('@routePatientActions');
+      .wait('@routePatientActions')
+      .wait('@routePatientFlows');
+
+    cy.wait(200);
```

Comment:
Pushing this up to see if it fixes things when cypress runs in circleci. 

Something is running out of sync. I think it's receiving the first ws notification before the subscription is created. So the app isn't ready to handle it yet. But only sometimes. When I run them locally, it's 50/50 if the test passes or not.

Is there something we're not `.wait()`ing for properly? I can't tell yet.